src/controllers/scoresController.ts

```diff
@@ -35,3 +42,24 @@ async function getScoresFromDB(gameIndex: number) {
     throw err;
   }
 }
+
+function getCurrentTime(): string {
+  const today = new Date();
+  const date = `${today.getMonth()}-${today.getDate()}-${today.getFullYear()}`;
+  const time = `${today.getHours()}:${today.getMinutes()}:${today.getSeconds()}`;
+  return `${date} ${time}`;
+}
```

Comment:
`return new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '')` gives the same result. https://stackoverflow.com/questions/10645994/how-to-format-a-utc-date-as-a-yyyy-mm-dd-hhmmss-string-using-nodejs

Reply:
Done in 299ea2f